index.html

```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
-      - Teymur Kheirkhabarov, <a href="https://twitter.com/HeirhabarovT">@HeirhabarovT</a> (<a href="https://www.bi.zone">BI.ZONE SOC</a>) 🇷🇺 <br>
-      - Daniel Bohannon, <a href="https://twitter.com/danielhbohannon">@danielhbohannon</a> (<a href="https://www.fireeye.com/services.html">FireEye</a>) 🇺🇸 <br>
-      - Alexey Potapov (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) 🇷🇺 <br>
-      - Kirill Kiryanov (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) 🇷🇺 <br>
-      - Egor Podmokov (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) 🇷🇺 <br>
-      - Anton Kutepov (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) 🇷🇺 <br>
-      - Alexey Lednyov (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) 🇷🇺 <br>
-      - Anton Tyurin (<a href="https://www.ptsecurity.com/ww-en/">PT ESC</a>) 🇷🇺 <br>
-      - Jan Hasenbusch (<a href="https://www.bsi-fuer-buerger.de">BSI</a>) 🇩🇪 <br>
-      - Eva Maria Anhaus (<a href="https://www.bsi-fuer-buerger.de">BSI</a>) 🇩🇪 <br>
-      - Diego Perez, <a href="https://twitter.com/darkquassar">@darkquassar</a> (Independent Researcher) 🇦🇷 <br>
-      - Mikhail Larin (<a href="https://csirt.jet.su">Jet CSIRT</a>) 🇷🇺 <br>
-      - Alexander Akhremchik (<a href="https://csirt.jet.su">Jet CSIRT</a>) 🇷🇺 <br>
-      - Dmitriy Lifanov (<a href="https://csirt.jet.su">Jet CSIRT</a>) 🇷🇺 <br>
-      - Alexey Balandin, <a href="https://twitter.com/Kriks87">@Kriks87</a> (<a href="https://csirt.jet.su">Jet CSIRT</a>) 🇷🇺 <br>
-      - Roman Rezvukhin (<a href="https://www.group-ib.com/cert.html">CERT-GIB</a>) 🇷🇺 <br>
-      - Alina Stepchenkova (<a href="https://www.group-ib.com/cert.html">CERT-GIB</a>) 🇷🇺 <br>
-      - Timur Zinniatullin (<a href="https://www.angaratech.ru/">Angara technologies group</a>) 🇷🇺 <br>
-      - Gleb Sukhodolskiy (<a href="https://www.angaratech.ru/">Angara technologies group</a>) 🇷🇺 <br>
-      - Victor Sergeev, <a href="https://twitter.com/stvetro">@stvetro</a> (<a href="https://www.helpag.com">Help AG</a>) 🇦🇪 <br>
-      - Ilyas Ochkov, <a href="https://t.me/CatSchrodinger">@CatSchrodinger</a> (Independent Researcher) 🇷🇺 <br>
-      - James Pemberton, <a href="https://twitter.com/4A616D6573">@4A616D6573</a> (<a href="https://www.hydro.com.au">Hydro Tasmania</a>) 🇦🇺 <br>
-      - Denis Beyu (<a href="https://citto.ru">GKU TO CITTO</a>) 🇷🇺 <br>
-      - Mateusz Wydra, <a href="https://twitter.com/sn0w0tter">@sn0w0tter</a> (<a href="https://www.relativity.com/ediscovery-software/relativityone/trust/threat-prevention/our-team/">Relativity</a>) 🇵🇱 <br>
-      - Jakob Weinzettl, <a href="https://twitter.com/mrblacyk">@mrblacyk</a> (<a href="https://www.tieto.com/en/what-we-do/managed-services-and-integration/security-services/security-operations-center/">Tieto SOC</a>) 🇵🇱 <br>
+      <h2>Previous sprints participants</h2>
+      <h4>Every sprint you earn one &#11088</h4>
```

Comment:
Stars look great! 
Could you please also put companies/"independent researcher" descriptions for all the members as we did it before?

Reply:
I think it also makes sense to sort the participants by the amount of stars in the list.

Reply:
> Could you please also put companies/"independent researcher" descriptions for all the members as we did it before?
Sure, I'll do it, thanks!

> I think it also makes sense to sort the participants by the amount of stars in the list.
Again with the sorting? ((: Ok, I'll do it.

Reply:
https://github.com/oscd-initiative/oscd/pull/4/commits/dc07dae46a9ae22f8837b26e8decfab8f0c3f4be

Reply:
> Every sprint you earn one

Could you please change it to something neutral yet informative, i.e.:

```
Stars reflect the number of sprints participated.
```

Current wording harshly pushes a reader to a position of a person that really needs the stars. Which is not always the case

Reply:
Done, thanks for the advice!